static-assets/components/cstudio-forms/controls/rte.js

```diff
@@ -398,7 +399,7 @@ CStudioAuthoring.Module.requireModule(
                 ? 'dark'
                 : 'default',
             external_plugins: external,
-
+            paste_remove_styles_if_webkit: false,
```

Comment:
By setting this prop to false, styles are allowed when pasted, but still filtered only to allow text-decoration (for strikethrough and underline).